gator/comments.py

```diff
@@ -34,3 +35,10 @@ def count_multiline_java_comment(contents):
     pattern = re.compile(MULTILINECOMMENT_RE, re.MULTILINE)
     matches = pattern.findall(contents)
     return len(matches)
+
+
+def count_multiline_python_comment(contents):
+    """Counts the number of multiline Python comments in the code"""
+    pattern = re.compile(MULTILINECOMMENT_RE_PYTHON, re.DOTALL)
```

Comment:
This needs `re.MULTILINE`, and `re.DOTALL` should be removed -- that enables `.` substituting for new lines, which you do not want with your current regex; you could change the regex to enable that however, if you wanted.